packages/charts/react-charting/src/components/ResponsiveContainer/ResponsiveContainer.tsx

```diff
@@ -70,19 +70,37 @@ export const ResponsiveContainer: React.FC<IResponsiveContainerProps> = props =>
     }
 
     return React.Children.map(props.children, child => {
+      const commonStyles = {
+        root: {
+          ...child.props.styles?.root,
+          width: '100%',
+          height: '100%',
+        },
+        chartWrapper: {
```

Comment:
>chartWrapper: { [](http://example.com/codeflow?start=8&length=15)

Does this work fine if someone does not use Responsive container

Reply:
These styles will only be applied when ResponsiveContainer is used.